MAUI/Calendar/views.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: post
+title: Views in MAUI Calendar widget | Syncfusion
+description: Learn here about the Views feature of Syncfusion MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: Calendar
+documentation: ug
+---
+
+
+# Multiple Calendar Views in MAUI (SfCalendar)
+The `SfCalendar` widget has four Calendar views to display. It can be assigned to the widget by using the `View` property. `Month` view is initially rendered by Default. The current date will be displayed initially for all the Calendar views.
+
+## Month view
+The Month view displays the current month days, and usually a few days of previous and next month. By default, initially displays the current month dates and the current date is highlighted by a separate color different from the rest of the dates color in `Month` view.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+{% include_relative code-snippet/month-view.xaml %}
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.View = CalendarView.Month;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view in .NET MAUI Calendar.](images/views/maui-month-view.png)
+
+### Number Of Visible Weeks view
+In the month view, number of visible weeks can be customized by using the `NumberOfVisibleWeeks` property in the Calendar. By default, Month view displays with the NumberOfVisibleWeeks as `6`.
```

Comment:
The number of visible weeks in the month view can be customized by using the `NumberOfVisibleWeeks` property in the Calendar. By default, the Month view displays with the NumberOfVisibleWeeks as `6.`

Reply:
Content modified for the number of visible weeks view